Modules/Brand/Http/Services/BrandService.php

```diff
@@ -227,15 +229,11 @@ public function getShop(string $brandKey): array
         $brand->tools_used = $brand->tools_used != '' ? explode(',', $brand->tools_used) : array();
         $brand->tag_shop_page = $brand->tag_shop_page != '' ? explode(',', $brand->tag_shop_page) : array();
 
-        //country
-        $country = Country::where('id', $brand->country)->first(['name']);
-        $brand->country = $country->name;
-        //headquarter
-        $headquarteredCountry = Country::where('id', $brand->headquatered)->first(['name']);
-        $brand->headquatered = $headquarteredCountry->name;
-        //shipped from
-        $productShippedCountry = Country::where('id', $brand->product_shipped)->first(['name']);
-        $brand->product_shipped = $productShippedCountry->name;
+        $countryIds = [$brand->country, $brand->headquatered, $brand->product_shipped];
```

Comment:
$brand->country, $brand->headquatered, $brand->product_shipped
are you sure these three contains Ids of the countries? because in whereIn you're checking the id.